homeassistant/components/websocket_api.py

```diff
@@ -262,6 +262,18 @@ def __init__(self, hass, request):
         self._handle_task = None
         self._writer_task = None
 
+    @property
+    def user(self):
+        """Return the user associated with the connection."""
+        return self.request.get('hass_user')
+
+    def context(self, msg):
```

Comment:
`msg` seems unused, change to property?

Reply:
I want to future proof it. Making a breaking change is a pain, especially for custom components.